pkg/util/subpath/subpath.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```

Comment:
Wrong year (applies to all new files in this PR?)

Reply:
Yeah, I wasn't exactly sure for this, as all the code is cut and paste from `/pkg/util/mount`. In that case, thought maybe it was better to keep copyright for the oldest code in the file. So I kept the year from whatever Linux/Windows file I was bringing the code over from. I'm still not sure what the right policy for that is.

Reply:
Just as a final clarification, now that most of the files that are in `pkg/volume/util/subpath` are originally copied from `pkg/util/mount` and preserve git history, I only set the Copyright date on the actually new file, which is `subpath.go`.